src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -128,11 +128,14 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
                   toggleOffThirdLevelMenu();
                 }}
                 className={clsx(
-                  'rounded-full border-[.1875rem] transition-all',
+                  'rounded-full border-[.1875rem] transition-all sm:hover:border-blue-400',
                   {
-                    'border-blue-400': openItemIndex === 0 && isTablet,
+                    'border-blue-400':
+                      (openItemIndex === 0 && isTablet) ||
+                      (openItemIndex === 0 && isSecondLevelMenuOpen),
```

Comment:
This condition shouldn't be added here, because if openItemIndex is not undefined then second level menu is always open. I guess that you should remove `&& isTablet` from the condition above